calendar_reboot.js

```diff
@@ -619,7 +621,7 @@ var Calendar = function(state) {
     //number of months we will need to be able to cover all the years the
     //user wants to track
     self.numberOfMonths = self.endDate.diff(self.startDate, 'months', true);
-    self.title = state.calendarTitle;
+    self.title = state.title;
```

Comment:
Why have this? It's a duplicate of `state.title`. You can always just do `calendar.state.title` if you want the title.